src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -2146,66 +2148,102 @@ namespace InternalHeatGains {
 						if ( errFlag ) ErrorsFound = true;
 					}
 
-					ZoneOtherEq( Loop ).SchedPtr = GetScheduleIndex( AlphaName( 3 ) );
+					std::string FuelTypeString( "" );
+					if ( AlphaName( 2 ) == "NONE" ) {
+						ZoneOtherEq( Loop ).OtherEquipFuelType = 0;
```

Comment:
Raw Integer?  How about a (class) enum?  This zero isn't meaningful to anyone else looking at this code.